sr/tools/bom/parts_db.py

```diff
@@ -1,16 +1,18 @@
 """Student Robotics parts database access library."""
 import csv
 
-import pkg_resources
 import six
+from pathlib import Path
 
 from sr.tools.bom import digikey, farnell, mouser, rs
 
+module_dir = Path(__file__).resolve().parent
+
 
 def get_db():
     """Get the parts DB that is embedded with this library."""
-    file = pkg_resources.resource_stream('sr.tools.bom', 'component_lib.csv')
-    return Db(six.StringIO(file.read().decode('UTF-8')))
+    file = module_dir / 'component_lib.csv'
```

Comment:
This creates a dependency on the package being on the filesystem in the same layout as the source, which the previous spelling (I think) did not. Is there no equivalent package resource lookup mechanism in the new world?

Reply:
Looks like [`importlib.resources`](https://docs.python.org/3/library/importlib.resources.html) might be what we want here? Not sure though.